models/repo_list_test.go

```diff
@@ -178,7 +178,7 @@ func TestSearchRepository(t *testing.T) {
 			count: 25},
 		{name: "AllPublic/PublicAndPrivateRepositoriesOfUserIncludingCollaborative",
 			opts:  &SearchRepoOptions{Page: 1, PageSize: 10, OwnerID: 15, Private: true, AllPublic: true, Template: util.OptionalBoolFalse},
-			count: 31},
+			count: 30},
```

Comment:
`count` decrease?

Reply:
Of course now we hide a public repo of a private org